src/Web/aspnet-core/src/Piggyvault.Application/Piggy/Accounts/AccountAppService.cs

```diff
@@ -120,19 +120,27 @@ public async Task<AccountEditDto> GetAccountForEdit(EntityDto<Guid> input)
         public async Task<ListResultDto<AccountPreviewDto>> GetAccountsAsync(GetAccountsAsyncInput input)
         {
             var output = new ListResultDto<AccountPreviewDto>();
+            var tenantId = 1;
```

Comment:
Will there be any case where tenant id can be null?

Reply:
Not for now